content/en/docs/application-store/external-apps/deploy-clickhouse.md

```diff
@@ -6,14 +6,15 @@ linkTitle: "Deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere"
 weight: 14340
 ---
 
-[ClickHouse](https://clickhouse.tech/) is a column-oriented database management system (DBMS) for online analytical processing of queries (OLAP). ClickHouse on QingCloud is a deeply customized ClickHouse cluster application maintaining ClickHouse cluster functions and featuring automated cluster management, data redistribution in clusters, and excellent performance with less cost.
+[ClickHouse](https://clickhouse.tech/) is a column-oriented database management system (DBMS) for online analytical processing of queries (OLAP). [RadonDB ClickHouse](https://github.com/radondb/radondb-clickhouse-kubernetes) is a deeply customized ClickHouse cluster application maintaining ClickHouse cluster functions and featuring automated cluster management, data redistribution in clusters, and excellent performance with less cost.
 
 This tutorial demonstrates how to deploy ClickHouse Operator and a ClickHouse Cluster on KubeSphere.
 
 ## Prerequisites
 
 - You need to enable [the OpenPitrix system](../../../pluggable-components/app-store/).
 - You need to create a workspace, a project, and two user accounts (`ws-admin` and `project-regular`) for this tutorial. The account `ws-admin` must be granted the role of `workspace-admin` in the workspace, and the account `project-regular` must be invited to the project with the role of `operator`. This tutorial uses `demo-workspace` and `demo-project` for demonstration. If they are not ready, refer to [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/).
+- You need to enable your KubeSphere cluster for the external access. For more information, see [Project Gateway](../../../project-administration/project-gateway/).
```

Comment:
If the ZH sentence prevails, I believe the EN sentence here should be: 
You need to enable [Project Gateway](../../../project-administration/project-gateway/) for your KubeSphere cluster.

Reply:
It is important to emphasize that KS project gateway enable the external access,  the updated sentence:

You need to enable the external access for the gateway in KubeSphere project. If they are not ready, refer to [Project Gateway](../../../project-administration/project-gateway/).

Reply:
I think `You need to enable the gateway in your project to provide external access. If...` would be better. What do you think?
BTW, try to avoid forced push because it is difficult to know what has been changed.

Reply:
good~~~
 
Final sentense:
`You need to enable the gateway in your project to provide external access. If ...`